fmriprep/run_workflow.py

```diff
@@ -123,27 +121,15 @@ def create_workflow(opts):
 
     run_uuid = strftime('%Y%m%d-%H%M%S_') + str(uuid.uuid4())
 
-    log_dir = op.join(settings['output_dir'], 'fmriprep', 'log', run_uuid)
-
     # Check and create output and working directories
     # Using make_folder to prevent https://github.com/poldracklab/mriqc/issues/111
     make_folder(settings['output_dir'])
     make_folder(settings['work_dir'])
-    make_folder(log_dir)
-
-    logger.addHandler(logging.FileHandler(op.join(log_dir, 'run_workflow')))
```

Comment:
It might be "enough", but by removing just this one line we are leaving out a bunch of code that does not do anything. We had this situation previously where there was code for setting up the logs, but one crucial line was missing and despite the appearance of logs being enabled no log files were produced.